src/test/java/uk/gov/hmcts/dts/fact/mapit/MapItServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.dts.fact.mapit;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyString;
+
+public class MapItServiceTest {
+
+    private final MapitValidator mockMapitValidator = Mockito.mock(MapitValidator.class);
+    private final MapItService mapItService = new MapItService(mockMapitValidator);
+
+    @Test
+    public void testValidatePostcodesSuccess() {
+        // Expect no strings to be returned if all checks have passed
+        String[] testPostcodesArray = {"M0", "MO5", "MO53", "MO533"};
+        Mockito.when(mockMapitValidator.postcodeDataExists(anyString())).thenReturn(true);
```

Comment:
The class name Mockito is used throughout this this test class. May be we can use import static Mockito so it can be hidden